### ISSUE TITLE
Initially 'aria-activedescendant" is empty, so this should be a WARNING not ERROR

### ISSUE THREAD

**Issue by ks-himanshub** [validator/validator#1587]
URL being validated or code to reproduce error: sorry can't provide URL for some confidentiality purpose

```
Bad value for attribute aria-activedescendant on element [input](https://html.spec.whatwg.org/multipage/#the-input-element): An ID must not be the empty string.
```

actually initially no option is selected in a list box so 'aria-activedescendant" will be obviously empty, it will only get the value when I select any option or navigate through up and down key.

**Comment by ks-himanshub** [comment]
Also wants to mention, I am using downshift.js for combobox

**Comment by j9t** [comment]
@ks-himanshub, do you still have some more context, like a minimum reproducible code sample?

When testing something like the following, I cannot reproduce.

```html
<!doctype html>
<title>test</title>
<input aria-activedescendant>
```

**Comment by sideshowbarker** [comment]
Closing for lack of response.
